packages/components/src/Chip/Chip.css

```diff
@@ -23,6 +22,15 @@
   transition: all var(--ease-both) var(--timing-quick);
   align-items: center;
 }
+div.chip {
+  padding: 0 var(--base-unit);
+}
```

Comment:
```suggestion
}

```

Reply:
I've made all these spacing updates, but I feel like we could probably write a few eslint rules to catch this one in the future. I'll make a note.

Reply:
@scotttjob that was my thought when doing the suggested change haha. Thanks for making a note - I also haven't really done eslint rule stuff so can put it side desk for myself too - whoever gets there first! 👍